### DISCUSSION TITLE
Pinia Cannot stringify a function

### DISCUSSION THREAD

**Discussion by JuliWolf** [nuxt/nuxt#21402] [Questions (Nuxt 2)]
Hello
I'm trying to connect pinia to nuxt2 project and always get a lot of warning like 
`Cannot stringify a function {name}`

This problems comes from devalue package which can not parse functions, so I change @pinia/nuxt build module to custom and use the following state to connect pinia to __NUXT__ object
`nuxtState.pinia = JSON.parse(JSON.stringify(pinia.state.value))`

but I still get such errors. What could be done to solve this?

**Comment by spraddles** [comment]
See this thread for a solution:
https://github.com/nuxt/nuxt/issues/20889
